src/ni/vsbuild/PipelineExecutor.groovy

```diff
@@ -2,8 +2,8 @@ package ni.vsbuild
 
 class PipelineExecutor implements Serializable {
 
-   static void execute(script, List<String> lvVersions, List<String> dependencies = []) {
-      def pipelineInformation = new PipelineInformation('veristand', lvVersions, dependencies)
+   static void execute(script, String nodeLabel, List<String> lvVersions, List<String> dependencies = []) {
```

Comment:
We should make nodeLabel  optional, since someone may not require any setup other than knowing the LabVIEW version to use.

- [ ] Make a second execute method that takes nodeLabel.

```
static void execute(script, String nodeLabel, List<String> lvVersions, List<String> dependencies = []) {
   def pipelineInformation = new PipelineInformation(nodeLabel, lvVersions, dependencies)
   pipelineInformation.printInformation(script)
   ...
}

static void execute(script, List<String> lvVersions, List<String> dependencies = []) {
   execute(script, '', lvVersions, dependencies)
}
```

- [ ] This will also require a change to `Pipeline.groovy` to check if **nodeLabel** is empty or null. Otherwise, at line 92, `script.node("{pipelineInformation.nodeLabel} && $lvVersion")` will not find a node.